Makefile.docker

```diff
@@ -12,9 +12,11 @@ image_manifest_file = build/docker/build_image_manifest
 build_image_tag_file = build/docker/image_tag
 build_image_tag = $(shell cat $(build_image_tag_file))
 
-build_image_name = weaveworks/eksctl-build:$(build_image_tag)
+build_image = weaveworks/eksctl-build
+build_image_name = $(build_image):$(build_image_tag)
 
-intermediate_container_name = eksctl-build-$(unique_tag)
+eksctl_image = weaveworks/eksctl
+eksctl_image_name = "$(eksctl_image):$${EKSCTL_IMAGE_VERSION}"
```

Comment:
>$${EKSCTL_IMAGE_VERSION}

Double `$`?

Reply:
Makefile magic `$$` = `$` so it ends up an env variable in the shell command

Reply:
ahaaa, thanks!